docs/sdks/uid2-sdk-ref-mobile.md

```diff
@@ -0,0 +1,212 @@
+# Developer Guide for UID2 Mobile SDKs
+
+As of 2023-07-15 SDK version:
+ * iOS V 0.2.0
+ * Android V 0.4.0
+
+## Features
+
+The UID2 Mobile SDKs are designed to manage UID2 Identity on behalf of
+iOS and Android apps. It enables UID2 Identity to be persisted across
+app lifecycles by securely storing it on a device via platform-native
+encryption tools.
+
+By default, the SDK automatically refreshes UID2 Identity based on
+expiration dates. However, you can disable this to allow implementing
+apps to manage the UID2 Identity lifecycle manually.
+
+## Open Source
+
+The following open source GitHub repositories are available for these
+SDKs:
+
+-   <https://github.com/IABTechLab/uid2-ios-sdk>
+
+-   <https://github.com/IABTechLab/uid2-android-sdk>
+
+## Installation
+
+Follow the applicable installation instructions:
+
+-   iOS
+-   Android
+
+## Operating System, API and Programming Language Requirements
```

Comment:
@bleege & @genwhittTTD : i find it strange that we don't mention these requirements which were stated in the sdk repo readmes. So i think we should add it here. Thoughts?

Reply:
I think it's fine to have this information here for the sake of completeness.  My only concern is that it's another place where this information is stored via hardcoding and is likely to get out of date.  Perhaps it'd be better to link to the requirements details in SDK Repo READMEs in this section instead?

Reply:
I would leave the information in the SDK Readme's as that's where developers are going to look first.  It's also easier for us to maintain as the info is stored closer to the code.  Have direct links to that documentation in the README via (not just the README itself) is ideal.  For example:

Good
https://github.com/IABTechLab/uid2-ios-plugin-google-ima#requirements

Not Good
https://github.com/IABTechLab/uid2-ios-plugin-google-ima#readme

Reply:
Ok i have added links to refer to the readme requirement sections as Brad suggested